src/timedate/timedated.c

```diff
@@ -939,6 +950,9 @@ static int method_set_ntp(sd_bus_message *m, void *userdata, sd_bus_error *error
         LIST_FOREACH(units, u, c->units)
                 u->path = mfree(u->path);
 
+        if (set_size(c->set_ntp_calls) >= SET_NTP_IN_FLIGHT_MAX)
+                return sd_bus_error_set_errnof(error, EAGAIN, "Too many calls in flight.");
```

Comment:
hmm, this will cause this method to return EAGAIN, which is a bit icky, since that is typically a local transient error, that suggests "retry later" locally. But from our pov this is not like that though, when we let the call fail here, we let it fail for good.

moreover sd_bus_error_set_errnof() is not a function you really should be using unless you actually received an error number from somewhere else. otherwise just return a proper dbus error.

hence, I'd either drop this (because broker already enforces a limit here), or make it return a proper dbus error.